tutorials/get-started-deploy-kapsule/index.mdx

```diff
@@ -0,0 +1,223 @@
+
+1. Create a file called `whoami-deployment.yaml` on your local machine:
+
+  ```
+  nano whoami-deployment.yaml
+  ```
+
+2. Paste the following code into the file, then save and exit:
+
+  ```
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+  name: mydeployment
+  labels:
+    app: mydeployment
+  spec:
+  replicas: 2
+  selector:
+    matchLabels:
+    app: mydeployment
+  template:
+    metadata:
+    labels:
+      app: mydeployment
+    spec:
+    containers:
+      - name: mycontainer
+        image: rg.fr-par.scw.cloud/videodemo/whoami:latest
+    imagePullSecrets:
+      - name: registry-secret
+  ```
+
+  Be sure to replace `rg.fr-par.scw.cloud/videodemo/whoami:latest` with the relevant path to where your `whoami` image is stored in your container registry.
+
+  <Message type="tip">
+
+  To better understand the different parts of the deployment yaml, here is some further information:
+
+    - **apiVersion** specifies which version of the Kubernetes API to use to create the object
+    - **kind** specifies the kind of object defined in this yaml file, here a deployment. Other Kubernetes objects we can define in yaml files include Pods, Secrets, Namespaces and more.
+    - **metadata** helps uniquely identify our deployment objet: we give it a name and a label.
+    - **spec shows** the desired state for the deployment (2 replicas) and the **selector** field defines how the Deployment finds which Pods to manage, in this case using our predefined label
+    - **template** labels the Pods with that same label, and tells them which container to run, and where to find the image for it. It also gives the pull secret needed for access.
+
+  As this is a very basic yaml file for a Kubernetes deployment, you could reuse the code and adapt it for other simple deployments by changing the registry and image information.
+
+  </Message>
+
+3. Run the following command to deploy the deployment on the cluster:
+
+  ```
+  kubectl apply -f whoami-deployment.yaml
+  ```
+
+4. Run the following command to list all the cluster's resources. If everything went well with the deployment, you should see that your pods are running:
+
+  ```
+  kubectl get all
+  ```
+
+5. Run the following command, replacing the pod ID with an appropriate output from the previous command, to check that the application is running and listening on a port:
+
+  ```
+  kubectl logs pod/mydeployment-5599cbcb56-x6lb8
+  ```
+
+## Exposing the service for testing via NodePort (optional)
+
+Our application is up and running, and we could just stop at this point. However, we can carry out a few more steps to expose the port it’s running on so that we can go to the cluster's endpoint and check that the application is printing out its container ID as it is designed to do. We will achieve this via NodePort, a Kubernetes service which opens a port on every node of the cluster. Any traffic the cluster receives on this node is forwarded.
```

Comment:
```suggestion
Our application is up and running, and we could just stop at this point. However, we can carry out a few more steps to expose the port it is running on so that we can go to the cluster's endpoint and check that the application is printing out its container ID as it is designed to do. We will achieve this via NodePort, a Kubernetes service which opens a port on every node of the cluster. Any traffic the cluster receives on this node is forwarded.
```